veps/vep-2.md

```diff
@@ -0,0 +1,29 @@
+# VEP-2 Vitess Release Cycle
+
+```
+VEP: 1
+Title: Vitess Patch Releases
+Author: Deepthi Sigireddi
+Reviewers: Morgan Tocker, Rafael Chacon Vivas, Guido Iaquinti
+Status: Proposed
+Created: 2020-06-17
+```
+
+## Abstract
+
+VEP-1(#1) established a release cycle for Vitess. Now that there have been 3 major releases, this VEP proposes some changes to the release cycle based on that experience.
+All other sections of VEP-1 will continue to be in force without any change.
+
+## Versioning and Release Cadence
+
+The proposed release cycle is best described as _Monotonic Versioning_ with a release cadence similar to Kubernetes. The terminology `COMPATIBILITY.RELEASE` used in this document is directly sourced from the [Monotonic Versioning Manifesto](http://blog.appliedcompscilab.com/monotonic_versioning_manifesto/). Monotonic versioning itself is a modified version of [Semantic Versioning](https://semver.org/).
```

Comment:
@morgo I could use some help on getting this language right. If what we are proposing is no longer monotonic, I'll need to rewrite this section.

Reply:
From my reading of that link, we're not actually doing monotonic versioning either. I'd just leave the doc as is and change the week count

Reply:
I still had to address the patch release numbering. Right now we are doing releases like 6.0.YY-yyyymmdd. We want to do 6.0.1, 6.0.2 etc.